### PR TITLE
Sets ca_cert for replication remotes.

### PR DESCRIPTION
**Pull request by dkliban**
fixes: #4219

### CODE REVIEW

**Comment by dkliban** [comment]
I am writing a test for this in pulp_file.

File: pulpcore/app/replica.py [hunk 1]
```diff
@@ -78,6 +80,11 @@ def remote_extra_fields(self, upstream_distribution):
         return {}
 
     def create_or_update_remote(self, upstream_distribution):
+        ca_cert = None
+        if "PULP_CA_BUNDLE" in os.environ and os.environ["PULP_CA_BUNDLE"]:
+            with open(os.environ["PULP_CA_BUNDLE"], "r") as ca_cert_file:
+                ca_cert = ca_cert_file.read()
```
[the remark this answers is not in the document]
Comment (dkliban) [reply pulpcore/app/replica.py:86]: This environment variable is used to pass the `ca_cert` to the pulp-glue library that is used to talk to the upstream pulp. 

The other option is to change the Replicator.__init__() to accept a ca_cert string. Then the replication task can pass the value it finds on the UpstreamPulp.

[the remark this answers is not in the document]
Comment (dkliban) [reply pulpcore/app/replica.py]: I updated the implementation.

**Review by dralley — APPROVED** [review]
I like this implementation better, too.  Thanks.

**Review by ggainey — APPROVED** [review]
Looks sweet!

**Pull request merged**